pkg/components/components.go

```diff
@@ -14,11 +14,11 @@ import (
 const Name = "cluster-network-addons-operator"
 
 const (
-	MultusImageDefault            = "docker.io/nfvpe/multus:latest"
+	MultusImageDefault            = "quay.io/kubevirt/cluster-network-addon-multus:v3.2-1.gitbf61002"
 	LinuxBridgeCniImageDefault    = "quay.io/kubevirt/cni-default-plugins:v0.1.0"
 	LinuxBridgeMarkerImageDefault = "quay.io/kubevirt/bridge-marker:0.1.0"
 	SriovDpImageDefault           = "quay.io/booxter/sriov-device-plugin:latest"
-	SriovCniImageDefault          = "docker.io/nfvpe/sriov-cni:latest"
+	SriovCniImageDefault          = "quay.io/kubevirt/cluster-network-addon-sriov-cni:v1.1.0.git9e4c973"
```

Comment:
this one is missing release number, why?

Reply:
sorry I don't understand this is not the release `v1.1.0.git9e4c973` ?

Reply:
I think it should be either `registry/image-name:vX.Y.Z` for released images, or `registry/image-name:vX.Y.Z-1.gitXXXXXXX` for unreleased. Version is the last released version, release number (1) will be bumped once we push a newer unreleased image and the git ID is there for a simple way to identify used sources.

Reply:
ok no problem can you take another look now?

Reply:
Looks great, thank you